cocos/primitive/transform.ts

```diff
@@ -69,9 +69,9 @@ export function translate (geometry: IGeometry, offset: { x?: number; y?: number
  * @param value @zh 缩放量。@en The scaling size
  */
 export function scale (geometry: IGeometry, value: { x?: number; y?: number; z?: number }) {
-    const x = value.x || 0;
-    const y = value.y || 0;
-    const z = value.z || 0;
+    const x = value.x ?? 1.0;
```

Comment:
Potential breaking change, but more like a fix.

Reply:
Please describe breaks in code sample and output difference

Reply:
Done